package.json

```diff
@@ -4,6 +4,7 @@
   "license": "GPL-3.0",
   "scripts": {
     "lint": "tfm-lint --plugin -d /webpack",
+    "lint:custom": "eslint ./webpack",
```

Comment:
Just a nitpick, but why we don't call it `lint:foreman` since it seems we want to use Foreman rules?

Reply:
lint:custom will run anything that is in the `plugins` list. which happens to be now the foreman rules.
In foreman core the same command was called spelling, and I had to rename it since now it checked spelling and ouia-ids.
So I want to avoid re-naming things later